app/controllers/api/v1/feed_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class FeedController < Api::V1::ApiController
+      def index
+        start = params[:start] ? Time.parse(params[:start]) : Time.now
+        with_include = params[:include] ? true : false
```

Comment:
What happen if they send it as `'false'`?

Reply:
default value is false

Reply:
I mean, if they send the string `'false'`